frontend/public/components/catalog-items.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+
+import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
+import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
+import {normalizeIconClass} from './utils/icon';
+
+const CatalogItem = ({obj}) => {
+  return <CatalogTile
+    id={obj.resource.metadata.uid}
+    key={obj.resource.metadata.uid}
+    title={obj.tileName}
+    iconImg={obj.tileImgUrl}
+    iconClass={obj.tileIconClass ? `icon ${normalizeIconClass(obj.tileIconClass)}` : null}
+    vendor={obj.tileProvider ? `Provided by ${obj.tileProvider}` : null}
+    description={obj.tileDescription}
+  />;
+};
+
+CatalogItem.displayName = 'CatalogItem';
+
+CatalogItem.propTypes = {
+  obj: PropTypes.shape({
```

Comment:
We call Kubernetes objects `obj` elsewhere. This is a bit deceptive since it's not actually a Kubernetes obj, but our own wrapper.

I suggest s/obj/item and s/resource/obj

Reply:
fixed